ModularTegustation/fishing/code/fishcooking/fishmachines/fish_cookers.dm

```diff
@@ -0,0 +1,41 @@
+//Used to fry fish
+
+//You can fry Whitefish, Pink fish and shrimp.
+/obj/structure/fishcooker/fishfrier
+	name = "fish frier"
+	desc = "A machine to fry fish. Don't get any ideas."
+	icon_state = "fishfrier"
+	on_icon = "fishfrier_on"
+	cook_time = 8 SECONDS
+	cookables = list(
+		/obj/item/food/freshfish/white 						= /obj/item/food/fishfingers,
```

Comment:
what are those spacings?

Reply:
Lines them up, that's why

Reply:
![image](https://github.com/vlggms/lobotomy-corp13/assets/77302679/f6469de0-c62f-4873-82bc-3ae4a44f82cf)
Looks fine in the preview?